index.html

```diff
@@ -93,21 +93,24 @@ <h1>Projects</h1>
 					<div class="intro-text view">
 						<h1>Contact</h1>
 						<section class="contact-info">
-							<div>
-								phone: (907)-978-3238
-							</div>
-							<div>
-								email: holladayian@gmail.com
-							</div>
-							<div>
-								GitHub: @holladayian
-							</div>
-							<div>
-								LinkedIn: Ian Holladay
-							</div>
-							<div>
-								portfolio: Link
-							</div>
+							<table>
+								<tr>
+									<th>Phone</th>
+									<th>(907)-978-3238</th>
+								</tr>
```

Comment:
Tables are pretty cool, although I don't see how their too much different from divs with the right css